api/server/router/network/network.go

```diff
@@ -30,7 +30,6 @@ func (r *networkRouter) initRoutes() {
 	r.routes = []router.Route{
 		// GET
 		router.NewGetRoute("/networks", r.getNetworksList),
-		router.NewGetRoute("/networks/", r.getNetworksList),
```

Comment:
I don't think this one should be removed; this was added to address https://github.com/moby/moby/issues/24595 (in https://github.com/moby/moby/pull/24634).

I do see I didn't add a regression test in that PR; let me open a PR for that

Reply:
ok, i get it ;

Reply:
What I usually do in these cases, is using the "git blame" view on GitHub to see what commit changed the line, for example, in this case: https://github.com/moby/moby/blame/b9a3a51ac562b40f658645db1a34bedb894edddf/api/server/router/network/network.go#L33

From that, you can see the commit details https://github.com/moby/moby/commit/6ad4bf0a533f8851b0ddccb56fe3b457819f6146, which describes why the change was made, and also has a link to the pull request related to it;

<img width="446" alt="Screenshot 2020-11-30 at 12 59 25" src="https://user-images.githubusercontent.com/1804568/100607912-eab02000-330b-11eb-9511-634716d66274.png">